filebeat.config:
  modules:
    path: ${path.config}/modules.d/*.yml
    reload.enabled: false

# This has a problem for me:
# https://discuss.elastic.co/t/how-do-i-fix-invalid-fieldreference-errors/283912
# related to the traefik let's encrypt label
# ... and apparently it is not fixable?
#
#filebeat.autodiscover:
#  providers:
#    - type: docker
#      hints.enabled: true

filebeat.inputs:
- type: log
  paths:
  - /seedbox/radarr/logs/*.txt
  tags: ["radarr"]
- type: log
  paths:
  - /seedbox/sonarr/logs/*.txt
  tags: ["sonarr"]
- type: log
  paths:
  - /seedbox/lidarr/logs/*.txt
  tags: ["lidarr"]
- type: log
  paths:
  - /seedbox/cops/log/nginx/*.log
  - /seedbox/cops/log/php/*.log
  tags: ["cops"]
- type: log
  paths:
  - /seedbox/flexget/*.log
  tags: ["flexget"]
- type: log
  paths:
  - /seedbox/heimdall/logs/*/*.log
  tags: ["heimdall"]
- type: log
  paths:
  - /seedbox/hydra2/logs/*
  tags: ["hydra2"]
- type: log
  paths:
  - /seedbox/jackett/Jackett/log.txt
  tags: ["jackett"]
- type: log
  paths:
  - /seedbox/plex/Library/Application Support/Plex Media Server/Logs/*
  - /seedbox/plex/Library/Application Support/Plex Media Server/Logs/*/*
  tags: ["plex"]
- type: log
  paths:
  - /seedbox/sabnzbd/logs/*
  tags: ["sabnzbd"]
- type: log
  paths:
  - /seedbox/tautulli/logs/*
  tags: ["tautulli"]
- type: log
  paths:
  - /seedbox/tentacles/log/*
  tags: ["tentacles"]

processors:
- add_cloud_metadata: ~

output.logstash:
  hosts:
    - logstash:5044
# output.elasticsearch:
#   hosts: elastic:9200
#   username: '${ELASTICSEARCH_USERNAME:}'
#   password: '${ELASTICSEARCH_PASSWORD:}'
